projects/addon-doc/package.json

```diff
@@ -15,9 +15,8 @@
     "license": "Apache-2.0",
     "dependencies": {
         "@angular-devkit/schematics": "12.2.18",
-        "highlight.js": "11.8.0",
-        "markdown-it": "12.3.2",
-        "ngx-highlightjs": "4.1.4",
+        "markdown-it": "13.0.1",
+        "ngx-highlightjs": "6.1.3",
```

Comment:
6 version of highlightjs is compatible with ng 16?

Reply:
if you upgrade to a higher version 7/8/9 then you will have error in console about that the minimum version should be angular 13